### PR TITLE
feat: limitation multi select from api

### PR DESCRIPTION
**Pull request by mdeljavan**

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -314,20 +291,11 @@ type SettingsNode implements Node {
...
 
-  """
-  The ID of the object.
-  """
+  """The ID of the object."""
   id: ID!
   phase: Phase!
   startText: String
   managerOverallReviewText: String
-  selfReviewSahabinessHelpModalText: String
```
Comment (mh-alahdadian) [inline schema.graphql:324]: what happens to this

File: src/shared/forminator/inputs/SelectMultiAutoComplete.tsx [hunk 2]
```diff
@@ -38,15 +38,20 @@ function SelectMultiAutoComplete<Suggestion extends BaseSuggestion = BaseSuggest
   label,
   textFieldOptions,
   excludes,
-  maximumValues = Number.POSITIVE_INFINITY,
-  messageMaximumReviewers = '',
+  maximumValues: maximumValuesProp = -1,
```
Comment (mh-alahdadian) [inline src/shared/forminator/inputs/SelectMultiAutoComplete.tsx:41]: why you not use `Number.POSITIVE_INFINITY` directly here

**Review by mh-alahdadian — CHANGES_REQUESTED** [review]

File: schema.graphql (hunk 1 above)
Comment (mdeljavan) [reply schema.graphql:324]: we need this

Comment (mh-alahdadian) [reply schema.graphql:324]: you removed all these lines 
they had no usage ?

File: src/shared/forminator/inputs/SelectMultiAutoComplete.tsx (hunk 2 above)
Comment (mdeljavan) [reply src/shared/forminator/inputs/SelectMultiAutoComplete.tsx:41]: Because if there is no limit. the back-end will send -1

**Review by mh-alahdadian — APPROVED** [review]

**Pull request merged**